Development/src/pages/devices/ChannelMappingMatrix.js

```diff
@@ -0,0 +1,514 @@
+import { useState } from 'react';
+import {
+    Table,
+    TableBody,
+    TableCell,
+    TableHead,
+    TableRow,
+    Tooltip,
+    Typography,
+} from '@material-ui/core';
+import {
+    ReferenceField,
+    ReferenceManyField,
+    SingleFieldList,
+} from 'react-admin';
+import get from 'lodash/get';
+import ChipConditionalLabel from '../../components/ChipConditionalLabel';
+import MappingButton from '../../components/MappingButton';
+import CollapseButton from '../../components/CollapseButton';
+
+const get_output_caps = (output_item, io) => {
+    return (
+        <>
+            <Typography variant="body1">{'Description:'}</Typography>
+            {output_item.properties.description}
+            <Typography variant="body1">{'Routable inputs:'}</Typography>
+            {output_item.caps.routable_inputs !== null
+                ? output_item.caps.routable_inputs
+                      .map(input_name => {
+                          return input_name === null
+                              ? 'MUTE'
+                              : get(io, `inputs.${input_name}.properties.name`);
+                      })
+                      .join(', ')
+                : 'null'}
```

Comment:
This `null` is "allows any input to be routed", need to think best way to indicate that.

Reply:
In the call we discussed writing this as ANY. I found the spec uses the phrase "no restrictions", so maybe that's also a possibility:

>If no such restrictions exist, the `routable_inputs` field MUST be set to `null`.
([link](https://amwa-tv.github.io/nmos-audio-channel-mapping/tags/v1.0.1/docs/4.0._Behaviour.html#output-routing-constraints))

Reply:
Similarly, will make this 'No Constraints' for initial merge.